### PR TITLE
Uyuni clconfguide ornela

### PR DESCRIPTION
**Pull request by 0rnela**
# Description

Adding corrections for better clarity.

# Target branches

Which documentation version does this PR apply to?

- [x] Master (Default)
- [ ] Manager-4.1
- [ ] Manager-4.0

# Links

Partial https://github.com/SUSE/spacewalk/issues/14503

### CODE REVIEW

File: modules/client-configuration/pages/registration-bootstrap.adoc [hunk 1]
```diff
@@ -40,7 +40,7 @@ If you register openSUSE Leap 15 or SLE 15 systems using Python 2, the bootst
 
 
 
-== Create a Bootstrap Script
+== Create a Bootstrap Script from Web UI
```
Comment (Loquacity) [inline modules/client-configuration/pages/registration-bootstrap.adoc:43]:
```suggestion
== Create a Bootstrap Script from {webui}
```

**Review by Loquacity — CHANGES_REQUESTED** [review]

File: modules/client-configuration/pages/registration-bootstrap.adoc [hunk 2]
```diff
@@ -103,7 +103,7 @@ cd /srv/www/htdocs/pub/bootstrap/
 cp bootstrap.sh bootstrap-sles12.sh
 cp bootstrap.sh bootstrap-sles15.sh
 ----
-. Open [path]``bootstrap-sles12.sh`` for modification.
+. Open [path]``bootstrap-sles15.sh`` for modification.
     Scroll down until you can see the text shown below.
     If ``exit 1`` exists in the file, comment it out by typing a hash or pound sign (&#35;) at the beginning of the line.
```
Comment (keichwa) [inline modules/client-configuration/pages/registration-bootstrap.adoc:108]: Does the XML character entity result in the wanted output?

File: modules/client-configuration/pages/registration-bootstrap.adoc [hunk 3]
```diff
@@ -140,10 +140,11 @@ cd /srv/www/htdocs/pub/bootstrap/
 . Run this command to execute the bootstrap script on the client; replace [systemitem]`` EXAMPLE.COM`` with the host name of your client:
 +
 ----
-cat bootstrap-sles12.sh | ssh root@EXAMPLE.COM /bin/bash
+cat bootstrap-sles15.sh | ssh root@EXAMPLE.COM /bin/bash
 ----
 The script downloads the required dependencies located in the repositories directory you created earlier.
 . When the script has finished running, you can check that your client is registered correctly by opening the {productname} {webui} and navigating to menu:Systems[Overview] to ensure the new client is listed.
+. If you used the script to register Salt client, you can open {productname} {webui} and navigate to menu:Salt[Keys] where you can accept the client key.
```
Comment (keichwa) [inline modules/client-configuration/pages/registration-bootstrap.adoc:147]: Wondering whether "Salt client" requires an article?

File: modules/client-configuration/pages/registration-webui.adoc [hunk 4]
```diff
@@ -53,5 +53,5 @@ To review a package EULA, open the package details page in the {webui}.
 [WARNING]
 ====
 To register and use {centos}{nbsp}6, {oracle}{nbsp}6, {rhel}{nbsp}6, or {sleses}{nbsp}6 clients, you need to configure the {productname} Server to support older types of SSL encryption.
-For more information about how to resolve this error, see ``Registering Older Clients`` at xref:client-configuration:tshoot-clients.adoc[].
+For more information about how to resolve this special use case, see ``Registering Older Clients`` at xref:client-configuration:tshoot-clients.adoc[].
```
Comment (keichwa) [inline modules/client-configuration/pages/registration-webui.adoc:56]:
```suggestion
For more information, see ``Registering Older Clients`` at xref:client-configuration:tshoot-clients.adoc[].
```
I think we can remove it without information loss.

File: modules/client-configuration/pages/registration-bootstrap.adoc (hunk 3 above)
Comment (0rnela) [reply modules/client-configuration/pages/registration-bootstrap.adoc:147]: We will have it in different PR.

File: modules/client-configuration/pages/registration-bootstrap.adoc [hunk 5]
```diff
@@ -105,7 +105,7 @@ cp bootstrap.sh bootstrap-sles15.sh
 ----
 . Open [path]``bootstrap-sles15.sh`` for modification.
     Scroll down until you can see the text shown below.
-    If ``exit 1`` exists in the file, comment it out by typing a hash or pound sign (&#35;) at the beginning of the line.
+    If ``exit 1`` exists in the file, comment it out by typing a hash sign at the beginning of the line.
```
Comment (Loquacity) [inline modules/client-configuration/pages/registration-bootstrap.adoc:108]: Americans tend to use 'pound'.

File: modules/client-configuration/pages/registration-bootstrap.adoc (hunk 3 above)
Comment (Loquacity) [inline modules/client-configuration/pages/registration-bootstrap.adoc:147]:
```suggestion
. If you used the script to register the Salt client, open the {productname} {webui} and navigate to menu:Salt[Keys] to accept the client key.
```

**Review by keichwa — APPROVED** [review]

**Review by Loquacity — APPROVED** [review]

**Pull request merged**